source-http-ingest/src/main.rs

```diff
@@ -0,0 +1,52 @@
+use anyhow::Context;
+use source_http_ingest::run_connector;
+use tokio::io;
+
+fn main() -> anyhow::Result<()> {
+    let runtime = start_runtime()?;
+
+    let stdin = io::BufReader::new(io::stdin());
+    let stdout = io::stdout();
+    let result = runtime.block_on(run_connector(stdin, stdout));
```

Comment:
I'm just generally curious about this: Is there an advantage to using `block_on` vs. `#[tokio::main]` and `.await`?

Reply:
The main reason for `block_on` is just so we can use `runtime.shutdown_background()`. This was actually cargo-culted over from [this bit in `connector-init`](https://github.com/estuary/flow/blob/f65a9d7f1a89f897b302f79bb89d53c7de682df3/crates/connector-init/src/main.rs#L42), though I'm pretty sure it would apply here as well.